Libraries/Components/Slider/Slider.js

```diff
@@ -197,6 +204,7 @@ const Slider = (
   forwardedRef?: ?React.Ref<typeof SliderNativeComponent>,
 ) => {
   const style = StyleSheet.compose(styles.slider, props.style);
+  const accessibilityState = props.accessibilityState || {};
```

Comment:
Does this need to be an empty object if no `props.accessibilityState`? It seems like SliderNativeComponent allows for null value for `accessibilityState` prop. In this approach you'd be a creating a non-used new object every time.

Reply:
Yeah I was wondering too, so should we just omit the empty object since  `props.accessibilityState` would return null if its empty

Reply:
Yea we can directly pass the prop to the native component.